x-pack/test/security_solution_endpoint_api_int/apis/endpoint_authz.ts

```diff
@@ -35,6 +35,7 @@ export default function ({ getService }: FtrProviderContext) {
   interface ApiCallsInterface {
     method: keyof Pick<typeof supertest, 'post' | 'get'>;
     path: string;
+    version?: string | number;
```

Comment:
Q: Do we need the `number` type at all?

Reply:
No, we don't. I initially (based on the spec) thought that `version` could be either a string or a number, but after looking at the implementation, looks like its just a number. I have made the changes already as part of the next batch of APIs.